samples/bluetooth/hap_ha/prj.conf

```diff
@@ -0,0 +1,58 @@
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_SMP=y
+CONFIG_BT_SMP_SC_PAIR_ONLY=y
+CONFIG_BT_SMP_MIN_ENC_KEY_SIZE=16
+
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_DEVICE_NAME="Hearing Aid sample"
+# Appearance: Generic Hearing aid (0x0A40)
+CONFIG_BT_DEVICE_APPEARANCE=2624
+
+CONFIG_BT_AUDIO=y
+CONFIG_BT_AUDIO_UNICAST_SERVER=y
+CONFIG_BT_ASCS_ASE_SNK_COUNT=1
+CONFIG_BT_ASCS_ASE_SRC_COUNT=1
+# Support an ISO channel per ASE
+CONFIG_BT_ISO_MAX_CHAN=2
+
+CONFIG_BT_PACS_SNK_CONTEXT=0x0047
+CONFIG_BT_PACS_SRC_CONTEXT=0x0003
```

Comment:
Similarly with the appearance, since this is a sample I guess it would be nice if we had a human-readable format as a comment for these values